client/src/main/java/hudson/plugins/swarm/Client.java

```diff
@@ -30,6 +30,8 @@
 
     private final Options options;
 
+    public static final int EXIT_CODE_PID_EXISTS = 29;
```

Comment:
Is there any specific reason to use 29 as the exit code? While there isn't any specific convention for exit codes, the number 29 does strike me as particularly arbitrary. I think I'd prefer to use the standard value of 1 for all exit codes. If the unit test needs to be able to differentiate this type of exit from another exit of status 1, I think it should read standard error.